This test tries to set a file date to 2108-12-30.
On illumos, the os.utime() call raises:
	OSError: [Errno 79] Value too large for defined data type
when the test expects an overflow error.

See also https://www.illumos.org/issues/17413

--- Python-3.14.6/Lib/test/test_zipfile/test_core.py.orig
+++ Python-3.14.6/Lib/test/test_zipfile/test_core.py
@@ -652,7 +652,7 @@
             self.skipTest(f'time.localtime({ts}) raises OverflowError')
         try:
             os.utime(TESTFN, (ts, ts))
-        except OverflowError:
+        except (OverflowError, OSError):
             self.skipTest('Host fs cannot set timestamp to required value.')
 
         mtime_ns = os.stat(TESTFN).st_mtime_ns
